src/components/landing-page/cd.landing-page.vue

```diff
@@ -73,7 +124,7 @@
 import { Component, Vue, toNative, Ref, Watch } from 'vue-facing-decorator';
 import { CzForm, CzNotifications, Notifications } from '@cznethub/cznet-vue-core';
```

Comment:
Instead of creating your own component, use:
https://github.com/cznethub/cznet-vue-core/blob/develop/src/components/cz.file-explorer.vue

Reply:
(unless you already connected with @Maurier while I've been gone in the last week and decided to take this approach)